.github/workflows/regression_test.yaml

```diff
@@ -57,8 +57,9 @@ jobs:
         run: python -m pip install torch torchvision torchao
       - name: Install remaining dependencies
         run: |
-          python -m pip install -e ".[dev]"
           python -m pip install lm-eval>=0.4.5
+          python -m pip install -e ".[dev]"
+          python -m pip install -e ".[async_rl]"
```

Comment:
Do we actually need this? I thought our regression test is basically a no-op at this point